lib/api/api.dart

```diff
@@ -626,35 +623,6 @@ class ApiSingleton {
     }
   }
 
-  Future<List<Topic>?> getTopicsSubscribed(String userIdentifier) async {
```

Comment:
so then, this should be remove too

`static const get_my_topics = '/topics_subscribed/';`